pkg/error/error_test.go

```diff
@@ -28,15 +28,15 @@ func TestGetErrorCount(t *testing.T) {
 			FilePath: "some/path",
 			Errors: []ValidationError{
 				{
-					LineNumber: 1,
+					LineNumber: 3,
```

Comment:
It should not replace `LineNumber` by `3` here.

Do you have an idea, why the bot does that? @klaernie